content/MBC1.md

```diff
@@ -8,17 +8,17 @@ types of MBCs.
 Note that the memory in range 0000-7FFF is used both for reading from
 ROM and writing to the MBCs Control Registers.
 
-## ROM/RAM access
+## Memory
 
 ### 0000-3FFF - ROM Bank 00/20/40/60 (Read Only)
 
-This area normally contains the first 16KBytes (bank 00) of the cartridge
+This area normally contains the first 16 KiB (bank 00) of the cartridge
 ROM. Can contain banks 20/40/60 in mode 1 (see below), or banks 10/20/30
```

Comment:
I would put a $ here to clarify that this is hex.

Reply:
Fixed